### PR TITLE
EncodeAndDecode.cs remove PSv2

### PR DESCRIPTION
**Pull request by CarloToso**
# PR Summary

Remove references to PSv2 in `EncodeAndDecode.cs`

## PR Context

#18323

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

File: src/System.Management.Automation/engine/remoting/server/serverremotesession.cs [hunk 1]
```diff
@@ -813,15 +811,6 @@ private void HandleCreateRunspacePool(object sender, RemoteDataEventArgs createR
                     if (value.ContainsKey("WSManStackVersion"))
                     {
                         var wsmanStackVersion = PSObject.Base(value["WSManStackVersion"]) as Version;
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/remoting/server/serverremotesession.cs:813]: It seems that `WSManStackVersion` is only used within this `if` statement, so shouldn't we delete lines 813 through 825?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/remoting/server/serverremotesession.cs (hunk 1 above)
Comment (CarloToso) [reply src/System.Management.Automation/engine/remoting/server/serverremotesession.cs]: Done

**Comment by CarloToso** [comment]
> @microsoft-github-policy-service agree [company="{}"]

**Comment by PaulHigin** [comment]
@CarloToso   My apologies.  We let this sit too long and another PR has made these same changes (#18795).  Since the change has already been made, I will close this as a duplicate.

**Pull request closed without merging**